understanding/22/consistent-help.html

```diff
@@ -36,86 +36,95 @@ <h2>Consistent Help</h2>
 
-        <p>When having problems completing a task on a Web site (or part of a Web site, what we call a <a>set of Web pages</a>), people with some types of disabilities may not be able to work through the issue without further help.  Issues could include difficulty:
-			completing a form, or finding a document or page which provides information required to complete a task.</p>
+         <p>The location in a smaller viewport may be different than in a larger viewport, but it is best if the mechanism or link is consistent across a set of web pages. A consistent location, both visually and programmatically, is the most usable.</p>
 
-        <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
+         <p>When having problems completing a task on a Web site (or part of a Web site, what we call a <a>set of Web pages</a>), people with some types of disabilities may not be able to work through the issue without further help.  Issues could include difficulty:
+         completing a form, or finding a document or page which provides information required to complete a task.</p>
 
-      <p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that when one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. It is also not the intent to require contact information if:</p>
-         <ul>
-            <li>A Web site is not supported,</li>
-				<li>Content is archived, or </li>
-            <li>When finding help would invalidate the activity, such as in a testing situation. </li>
-         </ul>
+         <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
 
-        <p>It is also not the intent of this Success Criterion to require a human be available at all times. If the human contact is not available during certain hours or certain days, information should be provided so the user can tell when it will be available. </p>
-   </section>
+         <p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages, these are not considered part of the "set of web pages" from which they are downloaded.</p>
```

Comment:
ed: the last sentence in this paragraph is a run-on. Consider replacing the comma between "pages" and "these" with either a sentence break or a semicolon.